### DISCUSSION TITLE
Don't copy custom attributes to hint

### DISCUSSION THREAD

**Discussion by tomfrit** [digitalfortress-tech/typeahead-standalone#75] [General]
I am trying to implement typeahead in a website with HTMX and AlpineJS. The problem is that the "hint" input is an exact clone of the original input, including all HTMX and AlpineJS attributes. Unfortunately, this doesn't really work and breaks typeaheads functionality (x-model on the input get copied and prevents typeahead from setting the hint). It would be great to either not copy special attributes at all or offer a configuration setting to select attributes to copy.
Thanks

**Comment by niketpathak** [comment]
closing this discussing as moved this to an issue.
